src/content/community/releases/_index.en.md

```diff
@@ -4,6 +4,40 @@ title = "Releases"
 weight = 40
 +++
 
+## v0.9.0
+
+* The gateway Pod has been renamed from `submariner` to `submariner-gateway`.
+* The Helm charts now support Submariner deployments using the Operator.
+* Broker creation is now managed by the Operator (and not `subctl`).
+* Each Submariner Pod now has its own service account with appropriate privileges.
+* For OpenShift, the Lighthouse CoreDNS server metrics are now included with the default CoreDNS metrics.
+* The `submariner_connections` metric is renamed to `submariner_requested_connections`.
+* For cases in which cross-cluster connectivity is provided without Submariner, `subctl` can now just deploy
+  Service Discovery via the `--components` flag.
+* Improved Service CIDR discovery for k3s deployments.
+* All Submariner metrics are now prefixed with `submariner_` in Prometheus.
+* With Globalnet deployments, Global IPs are now assigned to exported Services only. Previously, Globalnet annotated
+  every Service in the cluster, whether or not it was exported.
+* Lighthouse now allows a user to set name of CoreDNS custom ConfigMap.
+* The `strongswan` cable driver which was deprecated in v0.8.0 release is now removed.
+* Configuring Globalnet on a cluster is now optional with the `--globalnet` flag.
+* The Lighthouse-specific API is now removed in favor of [Kubernetes Multicluster Services API](https://github.com/kubernetes-sigs/mcs-api/).
+* A diagnostic tool [`subctl validate`](https://submariner.io/operations/deployment/subctl/#validate) was added to detect issues with the
+  Submariner deployment or some configurations in the cluster that may prevent Submariner from working properly.
+* Provision to notify user if `subctl` and `Submariner` versions are not matched in a setup was added.
+* Multiple new make-clean targets, `make clean-generated`, `make clean-clusters`and `make clean`, were added.
+* Submariner now supports automatic NAT discovery.
+* KIND environmet is now upgraded to version 0.10.
+* `version` flag of `make clusters` was renamed `k8s_version`.
+* New flags `repository` and `version` were added to `deploy-broker` command.
+* `submariner_service_import` and `submariner_service_export` Lighthouse metrics were added in Prometheus.
+* `make deploy` now supports enabling service discovery for deployments.
+* Lighthouse agent metrics are now included in Prometheus.
+* `service-discovery` flag of `deploy-broker` has been deprecated in favor of `components` flag. 
```

Comment:
I believe this is a dup of L15.

Reply:
L36 talks about deprecation and L15 talks about how to use --components flag. I can put L36 before L15

Reply:
yeah that would be fine.